project/src/components/offer-card/offer-card.tsx

```diff
@@ -1,39 +1,61 @@
-function OfferCard() : JSX.Element {
+import {Link} from 'react-router-dom';
+import {Fragment,useState} from 'react';
+import {Offers, Offer} from '../../types/offers';
+
+type OffersCardProps = {
+  offers: Offers,
+};
+
+function OfferCard(props: OffersCardProps): JSX.Element {
+  const {offers} = props;
```

Comment:
```suggestion
function OfferCard({offers}: OffersCardProps): JSX.Element {
```

можно использовать деструктуризацию сразу в параметрах.